### PR TITLE
[editorial] Change fallback effective directive to connect-src

### PR DESCRIPTION
**Pull request by antosart**

### CODE REVIEW

**Comment by antosart** [comment]
@mikewest this was suggested in this [comment](https://github.com/w3c/webappsec-csp/pull/611#issuecomment-1650136999) and makes sense to me.

File: index.bs [hunk 1]
```diff
@@ -4358,7 +4358,11 @@ this algorithm returns normally if compilation is allowed, and throws a
       ::
         1. Return `connect-src`.
 
-  4.  Return null.
+      : "`report`"
+      ::
+        1. Return null.
+
+  4.  Return `connect-src`.
```
Comment (mikewest) [inline index.bs:4365]: Perhaps add a note here that this is intended to be a default fallback for new fetch destinations that are added which don't explicitly fall into one of the other categories?

**Review by mikewest — APPROVED** [review]
LGTM. Sounds like a reasonable change (and we don't handle `null` well in the spec generally, so picking a directive is a good thing).

File: index.bs (hunk 1 above)
Comment (antosart) [reply index.bs:4365]: makes sense, done.

**Pull request merged**